jsonschema-validation.xml

```diff
@@ -740,6 +740,23 @@
                 </t>
             </section>
 
+            <section title='"readOnly"'>
+                <t>
+                    If it has a value of boolean true, this keyword indicates that the value of the
+                    instance is managed exclusively by the server or the owning authority, and
+                    attempts by a user agent to modify the value of this property are expected to be
+                    ignored or rejected by a server.
+                </t>
+                <t>
+                    For example, this property would be used to mark a server-generated serial
+                    number as read-only.
+                </t>
+                <t>
+                    The value of this keyword MUST be a boolean.
+                    The default value is false.
```

Comment:
After _much_ debate, we settled on this sort of language for default behavior:

> Omitting this keyword has the same behavior as a value of false.

Reply:
Addressed both remarks. Also move the type description in its own paragraph similarly with other keywords.